app/services/suggestions_service.rb

```diff
@@ -1,5 +1,6 @@
 class SuggestionsService
   def call
+    ::SequelRails::Railties::LogSubscriber.reset_count
```

Comment:
Since this is normally reset before the worker runs, it would be better to avoid this getting called here and the log line was potentially inaccurate - rather then resetting outside of a known lifecycle potentially leaving a point of confusion for other developers if this service ever ends up part of other jobs/actions in the future